Samples/resource-owner-password-credential/Angular2/openiddict-angular2-server/src/openiddict-angular2/Models/authModels.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.Mvc.ModelBinding;
```

Comment:
Please rename this file to match the C# coding guidelines (capital A on first letter)